libs/partners/milvus/langchain_milvus/utils/sparse.py

```diff
@@ -29,15 +29,18 @@ class BM25SparseEmbedding(BaseSparseEmbedding):
     https://milvus.io/docs/embed-with-bm25.md
     """
 
-    def __init__(self, corpus: List[str], language: str = "en"):
+    def __init__(self, corpus: List[str] = None, path: str = None, language: str = "en"):
```

Comment:
This is a breaking change use `*` to make and force it to be a last argument so that it's invoked by name.